enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/AuthProcedures.java

```diff
@@ -49,4 +49,29 @@ public void createUser( @Name( "username" ) String username, @Name( "password" )
         }
         shiroSubject.getUserManager().newUser( username, password, requirePasswordChange );
     }
+
+    @PerformsDBMS
```

Comment:
We have been looking for a better alternative ourselves. I'm sure we'll find one soon enough.